api/src/controllers/publications.js

```diff
@@ -192,8 +193,14 @@ async function getGoogleScholarPublications(req, res) {
   res.status(200).json(response);
 }
 
+/***
+ * The function performs the scraping logic to get the Google Scholar publications
+ * and returns it in a format that fits the publication model.
+ * @see models/publication.model.js
+ * @returns a publication
+ */
 async function scrapeGoogleScholar(url) {
-  console.log(playwrightConfig.gScholarHome + url);
+  logger.info(playwrightConfig.gScholarHome + url);
```

Comment:
Do we need both log statements (i.e.  the one on L159 and this one)?

Reply:
The one in line 203 is for the individual publication links while line 159 is for the profile link, which might be helpful for debugging the scraping logic for specific publications